### PR TITLE
Finishing navigation with Semantic UI

### PR DESCRIPTION
**Pull request by samigeorgiev**

### CODE REVIEW

File: client/src/components/Layout.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    return (
+        <Sidebar.Pushable style={{ minHeight: '100vh' }}>
+            <NavigationSidebar
+                isShown={isSidebarOpen}
+                onClose={() => setIsSidebarOpen(false)}
+            />
+            <Sidebar.Pusher
+                dimmed={isSidebarOpen}
+                style={{ minHeight: '100vh' }}
```
Comment (toniyanchev) [inline client/src/components/Layout.jsx:35]: Remove the spaces between the brackets

Comment (samigeorgiev) [reply client/src/components/Layout.jsx:35]: My React lint file force me to add space between curly and object

Comment (toniyanchev) [reply client/src/components/Layout.jsx:35]: Alright then

File: client/src/components/navigation/items/Search.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Search, Responsive } from 'semantic-ui-react';
+
+const Search_ = props => {
+    return (
+        <>
+            <Responsive minWidth={Responsive.onlyTablet.maxWidth}>
```
Comment (toniyanchev) [inline client/src/components/navigation/items/Search.jsx:8]: Is the logic correct on this line?

Comment (samigeorgiev) [reply client/src/components/navigation/items/Search.jsx:8]: What do you mean? Why is min width equals to Max Width?

Comment (toniyanchev) [reply client/src/components/navigation/items/Search.jsx:8]: That's what I mean

Comment (samigeorgiev) [reply client/src/components/navigation/items/Search.jsx:8]: Responsive.OnlyTablet.MaxWidth is the boundary where mobile dimensions start from, so Desktop styles should begin from there and Mobile ones therefore should end there

Comment (toniyanchev) [reply client/src/components/navigation/items/Search.jsx:8]: Everything's clear now!

**Pull request merged**